std_msgs/Header header
float32 rho
float32 theta
float32 cog_x
float32 cog_y
float32 area
float32 Z
float64[7] camera_pose
float64[80] marker_pose

